apps/aestratum/src/aestratum_config.erl

```diff
@@ -0,0 +1,195 @@
+-module(aestratum_config).
+
+-export([setup_env/1,
+         read_keys/1,
+         read_keys/2]).
+
+-import(aestratum_fn, [ok_val_err/2, is_ok/1, ok_err/2, val/3, key_val/2]).
+
+-include("aestratum.hrl").
+-include("aestratum_log.hrl").
+
+-define(DEFAULT_HOST, <<"pool.aeternity.com">>).
```

Comment:
To be amended cc @sennui ?

Reply:
I'll fix it.

Reply:
@skkw you mentioned that you changed that? maybe dropped out during some push?